deploy/crds/csi.ibm.com_v1_ibmblockcsi_cr.yaml

```diff
@@ -3,10 +3,56 @@ kind: IBMBlockCSI
 metadata:
   name: ibm-block-csi
   namespace: kube-system
+  labels:
+    app.kubernetes.io/name: ibm-block-csi-operator
+    app.kubernetes.io/instance: ibm-block-csi-operator
+    app.kubernetes.io/managed-by: ibm-block-csi-operator
 spec:
   controller:
     repository: ibmcom/ibm-block-csi-driver-controller
     tag: "1.0.0"
+    imagePullPolicy: IfNotPresent
+    affinity:
+      nodeAffinity:
+        requiredDuringSchedulingIgnoredDuringExecution:
+          nodeSelectorTerms:
+            - matchExpressions:
+                - key: kubernetes.io/arch
+                  operator: In
+                  values:
+                    - amd64
+
   node:
     repository: ibmcom/ibm-block-csi-driver-node
     tag: "1.0.0"
+    imagePullPolicy: IfNotPresent
+    affinity:
```

Comment:
maybe put comment and say that affinity will impact also the following side car container of the csi node. So it will be clear that other images below also impacted by this setting.,

agree?

Reply:
affinity is per pod, not per container, I don't think we should say that.

Reply:
Maybe we should add a comment to the controller and node like.
# Controller pod is created as a statefulSet with the ibm-block-csi-driver-controller and these sidecars(...)
# Node pod is created as daemonSet with the ibm-block-csi-driver-node and these sidecars (...)

Reply:
agree with Matt
please add some comment to givre context